app/Resources/views/page.html.twig

```diff
@@ -284,7 +284,7 @@
                         enableExperimentalNewNoteButton: true,
                         enableCleanOnboardingTheme: true,
                         theme: 'clean',
-                        usernameUrl: '{{ absolute_url(path('profile-no-id')) }}',
+                        usernameUrl: '{{ absolute_url(path('profile', {id: 'replace'})|replace({'replace': ''})) }}',
```

Comment:
three values for `replace` but only two are related, could use a disambiguation like `{id: 'placeholder'}|replace(...)`